pkg/investigations/investigation/errors.go

```diff
@@ -33,6 +33,28 @@ func (e AWSClientError) Error() string {
 	return fmt.Sprintf("could not retrieve aws credentials for %s: %s", e.ClusterID, e.Err.Error())
 }
 
+type RestConfigError struct {
+	ClusterID string
+	Err       error
+}
+
+func (e *RestConfigError) Unwrap() error { return e.Err }
+
+func (e RestConfigError) Error() string {
+	return fmt.Sprintf("could not create rest config for %s: %s", e.ClusterID, e.Err.Error())
+}
+
+type OCClientError struct {
+	ClusterID string
+	Err       error
+}
+
+func (e *OCClientError) Unwrap() error { return e.Err }
+
+func (e OCClientError) Error() string {
```

Comment:
nit: use pointer receivers for all functions
```suggestion
func (e * OCClientError) Error() string {
```